reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -0,0 +1,652 @@
+- **v7.1.0-preview.4**
+- Make the switch parameter -Qualifier not positional for Split-Path (#12960) (Thanks @yecril71pl!)
+- Resolve the working directory as literal path for Start-Process when it's not specified (#11946) (Thanks @NoMoreFood!)
+- Make -OutFile parameter in web cmdlets to work like -LiteralPath (#11701) (Thanks @iSazonov!)
+- **v7.1.0-preview.3**
+- Fix string parameter binding for BigInteger numeric literals (#11634) (Thanks @vexx32!)
+- **v7.1.0-preview.2**
+- On Windows, Start-Process creates a process environment with all the environment variables from
+  current session, using -UseNewEnvironment creates a new default process environment (#10830)
+  (Thanks @iSazonov!)
+- Do not wrap return result to PSObject when converting ScriptBlock to delegate
+  (#10619)
+- **v7.1.0-preview.1**
+- Use invariant culture string conversion for -replace operator (#10954) (Thanks @iSazonov!)
+
+### Engine Updates and Fixes
+
+- **v7.1.0-preview.rc.2**
+- Rename Get-Subsystem to Get-PSSubsystem and fix two related minor issues (#13765)
+- Add missing PSToken token table entries to fix the PSParser API (#13779)
+- Add additional PowerShell modules to the tracked modules list (#12183)
+- Fix blocking wait when starting file associated with a Windows application (#13750)
+- Revert PSNativePSPathResolution to being an experimental feature (#13734)
+- **v7.1.0-preview.rc.1**
+- Make fixes to ComInterop code as suggested by .NET team (#13533)
+- **v7.1.0-preview.7**
+- Initial work of the subsystem plugin model (for minimal powershell) (#13186)
+- Optimize GetSystemLockdownPolicy for non-lockdown scenarios (#13438)
+- **v7.1.0-preview.6**
+- Refactor command line parser to do early parsing (#11482) (Thanks @iSazonov!)
+- Add support for some .NET intrinsic type converters (#12580) (Thanks @iSazonov!)
+- Refresh and enable the ComInterop code in PowerShell (#13304)
+- **v7.1.0-preview.5**
+- Ensure assemblies listed in the module manifest FileList field are not loaded (#12968)
+- **v7.1.0-preview.4**
+- Ensure null-coalescing LHS is evaluated only once (#12667)
+- Fix path handling bug in PSTask (#12554) (Thanks @IISResetMe!)
+- Remove extra line before formatting group (#12163) (Thanks @iSazonov!)
+- Make module formatting not generate error with strict mode (#11943)
+- Adding more ETW logs to WSMan plugin (#12798) (Thanks @krishnayalavarthi!)
+- Restrict loading of amsi.dll to system32 folder (#12730)
+- **v7.1.0-preview.3**
+- Set correct PSProvider full name at module load time (#11813) (Thanks @iSazonov!)
+- **v7.1.0-preview.2**
+- Allow case insensitive paths for determining PSModulePath (#12192)
+- Add PowerShell version 7.0 to compatible version list (#12184)
+- Discover assemblies loaded by Assembly.Load(byte[]) and Assembly.LoadFile (#12203)
+- **v7.1.0-preview.1**
+- Revert the PRs that made DBNull.Value and NullString.Value treated as $null (#11648)
```

Comment:
Remove this - since the feature was reverted it does not apply to GA. Also, remove the related item that added the feature being reverted.